CONTRIBUTING.md

```diff
@@ -0,0 +1,111 @@
+# How to contribute
+
+Operator Marketplace is Apache 2.0 licensed and accepts contributions via GitHub pull requests. This document outlines some of the conventions on commit message formatting, instructions on how to set up a dev environment, and other resources to help get contributions into operator-marketplace.  
+
+## Setting up a dev environment
+
+- Fork the repository on GitHub
+- Clone the forked repository in your go path.
+- Get access to an Openshift cluster. The marketplace-operator is currently supported on Openshift 4.0. You can find instructions on installing an Openshift cluster [here](https://github.com/openshift/installer).
+- The Cluster-Version-Operator(CVO) manages the lifecycle of the marketplace-operator along with other internal Openshift components. Before deleting the marketplace-operator deployment, instruct CVO to stop managing the marketplace-operator 
+```
+$ oc apply -f examples/cvo.override.yaml
+``` 
+- Delete the marketplace-operator deployment
+```
+$ oc delete deployment openshift-marketplace -n marketplace-operator
+```
+- Install [Operator-SDK](https://github.com/operator-framework/operator-sdk).
+- Compile the marketplace-operator and start the operator in your dev environment
+```
+$ operator-sdk up local --namespace=openshift-marketplace --kubeconfig=<path-to-kubeconfig-file>  
+```
+
+### Testing changes to RBAC policy
+- If there's any modification in existing RBAC policies, the modifications need to be tested using a separate deployment. Easiest way to do this is to build an image locally using [this dockerfile](./Dockerfile), push the image to a registry, and replace the [image here](https://github.com/operator-framework/operator-marketplace/blob/master/deploy/operator.yaml#L23) with the newly built image. The operator can then be redeployed with the new changes.   
+
+```
+$ cd <path-to-operator-marketplace-repo>
+
+$ operator-sdk build . --tag=<REGISTRYHOST>/<USERNAME>/marketplace-operator
+
+$ docker push <REGISTRYHOST>/<USERNAME>/marketplace-operator 
+
+$ sed -i -e 's?quay.io/openshift/origin-operator-marketplace:latest?<REGISTRYHOST>/<USERNAME>/marketplace-operator?g' deploy/operator.yaml
+
+$ oc apply -f deploy/operator.yaml
+```
+
+## Marketplace-operator on vanilla Kubernetes
+
+- To set up marketplace-operator on vanilla Kubernetes, [install Operator-Lifecycle-Manager(OLM)](https://github.com/operator-framework/operator-lifecycle-manager/blob/master/Documentation/install/install.md) in a Kubernetes Cluster. 
+- `cd` into the marketplace-operator repository and install the marketplace-operator using 
+```
+$ kubectl apply -f deploy/upstream
+```
+- To delete the marketplace-operator deployment use 
+```
+$ oc delete deployment marketplace
+``` 
+
+### Testing changes to RBAC policy
```

Comment:
nit: Since this section is specific for testing RBAC on vanilla kube, can the header reflect this? I was scrolling through and thought this was the same as the section for OpenShift, but this is specific for kubernetes. On a quick scroll it's tough to distinguish the two sections.

Reply:
@SamiSousa changed the title. Please let me know if it's easier to distinguish now.